pkg/api/index.go

```diff
@@ -95,6 +95,7 @@ func setIndexViewData(c *m.ReqContext) (*dtos.IndexViewData, error) {
 	if hasEditPermissionInFoldersQuery.Result {
 		children := []*dtos.NavLink{
 			{Text: "Dashboard", Icon: "gicon gicon-dashboard-new", Url: setting.AppSubUrl + "/dashboard/new"},
+			{Text: "Import", SubTitle: "Import dashboard from file or Grafana.com", Id: "import", Icon: "gicon gicon-dashboard-import", Url: setting.AppSubUrl + "/dashboard/import"},
```

Comment:
This results in multiple import menu items when both hasEditPermissionInFolders and org role admin/editor. The import menu link should always be rendered last in the menu.

![image](https://user-images.githubusercontent.com/1668778/41096085-cbac7542-6a53-11e8-9942-4bdb1de0330c.png)

Reply:
Yes, seems, missed this.